### PR TITLE
Mp 7066/update jupyter 1click

### PR DESCRIPTION
**Pull request by msannan2**

### CODE REVIEW

File: jupyter-conda-22-04/files/etc/systemd/system/jupyter-notebook.service [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+[Service]
+User=anaconda
+WorkingDirectory=/home/anaconda
+EnvironmentFile=/root/.digitalocean_passwords
+ExecStart=/home/anaconda/.conda/envs/jupyter/bin/jupyter lab --no-browser --IdentityProvider.token="${JUPYTER_TOKEN}" --ServerApp.allow_remote_access=true
+Restart=always
+SyslogIdentifier=jupyter-notebook
+
+[Install]
+WantedBy=multi-user.target
```
Comment (alexeigutium) [inline jupyter-conda-22-04/files/etc/systemd/system/jupyter-notebook.service:14]: github wants an empty line here

**Review by alexeigutium — APPROVED** [review]

File: jupyter-conda-22-04/files/etc/systemd/system/jupyter-notebook.service (hunk 1 above)
Comment (msannan2) [reply jupyter-conda-22-04/files/etc/systemd/system/jupyter-notebook.service]: Added

**Comment by scott** [comment]
there is a whitesource security check failing

**Review by scott — APPROVED** [review]

**Comment by msannan2** [comment]
> there is a whitesource security check failing

Yeah the whitesource check picked up some packages that had vulnerabilities in their older versions. This is not an issue since we are using the latest ones

**Pull request merged**